### ISSUE TITLE
Mediaprojection doesn't work with MediaRecorder

### ISSUE THREAD

**Issue by anonym24** [googlesamples/android-ScreenCapture#9]
Mediaprojection doesn't work with MediaRecorder

What am I doing wrong?

https://stackoverflow.com/questions/45266016/screen-recording-stop-failed-and-failure-delivering-result-resultinfo

Android 6 Marshmallow (Redmi 4 Prime, MIUI 8) http://www.gsmarena.com/xiaomi_redmi_4_prime-8263.php

**Comment by thagikura** [comment]
This is not an issue.

**Comment by anonym24** [comment]
@thagikura what is that then?

**Comment by thagikura** [comment]
Sorry, let me rephrase, this repository expects an issue of the sample. Please refer to other sites like Stackoverflow for usage questions.
